packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -163,6 +163,36 @@ const DashboardChartTile: FC<Props> = (props) => {
     >([]);
     const { user } = useApp();
 
+    type FilterPivot = {
+        field: string;
+        table: string;
+        value: string;
+    };
+    const getPivots = useCallback(
+        (data) => {
+            const keys = Object.keys(data);
+            const pivotDimension = savedQuery?.pivotConfig?.columns?.[0];
+            return keys.reduce((results: FilterPivot[], key) => {
+                const pivotKey = key.split('.');
```

Comment:
@ZeRego I'm not sure if you know a better way of extracting the pivots in here... I couldn't find anything similar in the code

Reply:
I don't know how well this is going to work. Specially for null/undefined/dates. The raw value of the pivot is available in the series object. 

See if there is any information on the series in the event object. If not we could change `packages/frontend/src/components/SimpleChart/index.tsx` -> `onChartContextMenu` to pass the relevant series as an argument. 

```onSeriesContextMenu(e, eChartsOptions.series[e.seriesIndex]);```